content/docs/rendering-elements.md

```diff
@@ -43,7 +43,7 @@ ReactDOM.render(
 );
 ```
```

Comment:
We can also replace the inline example above by embedding the new snippet too, like we do for the ["file" input type](https://reactjs.org/docs/forms.html#the-file-input-tag). 😁 

This is possible using the [`gatsby-remark-embed-snippet` plugin](https://www.gatsbyjs.org/packages/gatsby-remark-embed-snippet/)

Check out PR #323 for another example of this.